src/core/StelObject.hpp

```diff
@@ -332,7 +332,8 @@ public slots:
 	//! @param core
 	//! @param flags
 	//! @param decimals significant digits after the comma.
-	virtual QString getMagnitudeInfoString(const StelCore *core, const InfoStringGroup& flags, const int decimals=1) const;
+	//! @param magOffset magnitude offset to apply (can be due to changing distance)
```

Comment:
Can you please describe this new parameter in more detail? Or maybe it is described elsewhere?

Reply:
OK, this is final point who need to fixation

Reply:
I have described more. Basically when the distance of stars change because of proper motion/radial motion, magOffset is used to compute new apparant magnitude.